torch/_torch_docs.py

```diff
@@ -3082,7 +3082,10 @@ def merge_dicts(*dicts):
 
 .. math::
     y_{i} = 2^{x_{i}}
+
+.. note:: Alias for :func:`torch.special.exp2`.
```

Comment:
This is OK but eventually we should just replace these doc strings with the pointer to what they alias, like the arccos docs: https://pytorch.org/docs/master/generated/torch.arccos.html?highlight=alias.